turbo/stages/headerdownload/header_algos.go

```diff
@@ -892,24 +893,40 @@ func (hi *HeaderInserter) FeedHeaderPoW(db kv.StatelessRwTx, headerReader servic
 	}
 	// Calculate total difficulty of this header using parent's total difficulty
 	td = new(big.Int).Add(parentTd, header.Difficulty)
+
 	// Now we can decide wether this header will create a change in the canonical head
-	if td.Cmp(hi.localTd) > 0 {
-		hi.newCanonical = true
-		forkingPoint, err := hi.ForkingPoint(db, header, parent)
-		if err != nil {
-			return nil, err
+	if td.Cmp(hi.localTd) >= 0 {
+		reorg := true
+
+		// TODO: Add bor check here if required
```

Comment:
This is something which I am not sure of. If no other chain is using the PoW flow, I think we don't need to wrap this with `isBor` check. If not, we might have to change the function signature for checking if we're running bor.

Reply:
This PoW flow is only used for Polygon.